### ISSUE TITLE
SIGNATURES PROBLEMS FOR CCLE DATA

### ISSUE THREAD

**Issue by Tardins** [openbiox/UCSCXenaShiny#267]
Greetings!

When querying with signatures in Analyze Association between Gene (Signature) and Drug Response with CCLE Data, and Visualize Gene and Drug-Target Association with CCLE Data, I have had some problems. The shiny showns this message: 
Space is detected in your input. It's invalid. If you want to use genomic signature feature, please input a gene list.

I'm sure that I am using the genetic signatures correctly, as I can perform my analyzes in the other analysis categories of xena shiny

I would like to know if the analysis category Analyze Association between Gene (Signature) and Drug Response with CCLE Data, or
Visualize Gene and Drug-Target Association with CCLE Data allow the use of genetic signatures, or just analyze gene by gene, or this could just be a bug.

Regards,

Alan T.

**Comment by ShixiangWang** [comment]
Do you use the tool locally?

**Comment by ShixiangWang** [comment]
Or by the hiplot platform?

**Comment by ShixiangWang** [comment]
@Tardins 

Hi, T. XenaShiny on Hiplot is working now.

<img width="1171" alt="image" src="https://github.com/openbiox/UCSCXenaShiny/assets/25057508/9c7912d1-b398-47e1-b633-31d0b4192a46">


Thanks to @Miachol for maintaining it on hiplot.

**Comment by Tardins** [comment]
Hi, Thanks a lot for the feedback.
I was trying to use the hiplot platform, but I also tried using the R or Python environment. In both I got no results. However, I was using a different way to insert the signatures [ex: (TP53 + KRAS + H19)], and I see that you used a different way than mine. But I have already managed to carry out the analyzes I would like, using the way you demonstrated. Thanks a lot for the help.
